### ISSUE TITLE
[Slim] training ImageNet from scratch

### ISSUE THREAD

**Issue by redserpent7** [tensorflow/models#1173]
I am trying to prepare the date to train an ImageNet model from scratch and I am a bit confused about how the training works.

While preparing the TF records I noticed this file inside the Inception model data directory: "imagenet_metadata.txt". The file holds labels for 21842 classes yet the training script and "imagenet_lsvrc_2015_synsets.txt" file only works for 1000 classes.

I am wondering what modifications I need to do to train the model on the 21K classes not the 1K one?

**Comment by concretevitamin** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!

**Comment by redserpent7** [comment]
@concretevitamin I've already asked on SO. No one even commented on it

**Comment by redserpent7** [comment]
@concretevitamin here is the SO question if you care to answer there: [http://stackoverflow.com/questions/42816452/tensorflow-slim-imagenet-training](http://stackoverflow.com/questions/42816452/tensorflow-slim-imagenet-training)
